pandas/tests/scalar/test_timedelta.py

```diff
@@ -128,6 +128,61 @@ def test_unary_ops(self):
         assert abs(-td) == td
         assert abs(-td) == Timedelta('10d')
 
+    def test_mul(self):
+        # GH#19365
+        td = Timedelta(minutes=3)
+
+        result = td * 2
+        assert result == Timedelta(minutes=6)
+
+        result = td * np.int64(1)
+        assert isinstance(result, Timedelta)
+        assert result == td
+
+        result = td * 1.5
+        assert result == Timedelta(minutes=4, seconds=30)
+
+        result = td * np.array([3, 4], dtype='int64')
```

Comment:
I understand why this is, but I don't really like it. It would be much more friendly to return a TDI, maybe too magical though

Reply:
Yah I considered that too, came to the same conclusion.  It also would be troublesome if the other array wasn't 1D.

Reply:
is it easy to just return a TDI? this is very unituitive

Reply:
so you haven't answered my question here. this needs to return a TDI or raise TypeError. return a m8 array is simply not useful.